admin/tracking/class-tracking-settings-data.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Tracking
+ */
+
+/**
+ * Collects anonimized settings data.
+ */
+class WPSEO_Tracking_Settings_Data implements WPSEO_Collection {
+
+	/**
+	 * @var array ANONYMOUS_SETTINGS contains all of the option_names which need to be
+	 * anonimized before they can be sent elsewhere.
+	 */
+	const ANONYMOUS_SETTINGS = [
```

Comment:
Arrays in constants aren't supported in older versions of php. I am not sure if it's supported in PHP 5.6.  https://www.php.net/manual/en/migration56.new-features.php. I think PHP 5.6 supports it: php.net/manual/en/language.oop5.constants.php

Reply:
Good catch! Will change that.